Incompatible versions of node/npm

Is there a reason to use old versions of node here? Node 18 is the current LTS.

https://github.com/readthedocs/doc-diff/blob/main/package.json#L6-L9

I'm getting this error

```
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: readthedocs-doc-diff@1.0.0
npm ERR! notsup Not compatible with your version of node/npm: readthedocs-doc-diff@1.0.0
npm ERR! notsup Required: {"node":"^14.0.0","npm":"^7.0.0"}
npm ERR! notsup Actual:   {"npm":"9.5.0","node":"v18.15.0"}
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -27,12 +27,12 @@ commands:
 jobs:
   checks:
     docker:
-      - image: "cimg/node:14.20"
+      - image: "cimg/node:18.15"
     steps:
       - run-checks: {}
   build:
     docker:
-      - image: "cimg/node:14.20"
+      - image: "cimg/node:18.15"
     steps:
       - run-build: {}
 

--- a/package.json
+++ b/package.json
@@ -4,8 +4,8 @@
   "description": "Read the Docs doc visual diff extension",
   "main": "dist/doc-diff.js",
   "engines": {
-    "node": "^14.0.0",
-    "npm": "^7.0.0"
+    "node": "^18.15.0",
+    "npm": "^9.5.0"
   },
   "scripts": {
     "build": "webpack --mode=production --progress",

